server/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -304,8 +312,10 @@ public static Item forUpdate(String id,
                                      Symbol[] assignments,
                                      long requiredVersion,
                                      long seqNo,
-                                     long primaryTerm) {
+                                     long primaryTerm,
+                                     long sizeEstimate) {
             long usedBytes = SHALLOW_SIZE;
```

Comment:
minor: can we rename variable used here to `ITEM_SHALLOW_SIZE`?

I was going to comment that we already accounted for the shallow size in `ShardRequest.shallowSize` and later realized that this one is taking item's shallow size.

Reply:
Why exactly should we rename it?

Reply:
I also don't see the point of renaming it. But no strong opinion.